clijs/test/commands/config/get.test.ts

```diff
@@ -0,0 +1,21 @@
+import { describe, expect } from "vitest";
+import ConfigManager from "../../../src/common/config.js";
+import { ConfigKeys } from "../../../src/common/config.js";
+import { CliTest } from "../../setup.js";
+
+describe("config:get", () => {
+  CliTest("gets a value from the config file", async ({ cfgPath, runCommand }) => {
+    const { result } = await runCommand(["config", "get", ConfigKeys.RpcEndpoint]);
+    const configManager = new ConfigManager(cfgPath);
+
+    expect(result).to.equal(
+      configManager.getConfigValue(ConfigKeys.NilSection, ConfigKeys.RpcEndpoint),
```

Comment:
Aren't we expecting `undefined` here if we haven't set it previously? It seems to me that the `get` test should create (probably manually as writing a string to a file) a config of the expected content, and the test should check that we get that particular value (rather than just comparing the identity of the code inside the command with the same code outside it).